core/src/main/java/org/mobilitydata/gtfsvalidator/notice/InvalidEmailNotice.java

```diff
@@ -26,13 +26,15 @@
  *
  * <p><a href="https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md">GTFS
  * reference</a> does not provide any special requirements or standards.
- *
- * <p>Severity: {@code SeverityLevel.ERROR}
  */
 public class InvalidEmailNotice extends ValidationNotice {
 
   public InvalidEmailNotice(
-      String filename, long csvRowNumber, String fieldName, String fieldValue) {
+      String filename,
+      long csvRowNumber,
+      String fieldName,
+      String fieldValue,
+      SeverityLevel severityLevel) {
```

Comment:
We might want yo document via javadocs why this second constructor is needed.

Reply:
Done.